database/users_models.py

```diff
@@ -57,7 +57,7 @@ class Permission(db.Model):
 # Define a user table to store the user's data when they sign up for the saving challenges algorithm
 class UserInformation(db.Model):
```

Comment:
This model name will possibly need to change since some error message say user profile, and some say user information.